scripts/build/appveyor/configure_appveyor_VS2017.bat

```diff
@@ -11,13 +11,13 @@ cmake .. -G "Visual Studio 15 2017 Win64"
 -DCONTACT_STRUCTURAL_MECHANICS_APPLICATION=ON                                   ^
 -DSWIMMING_DEM_APPLICATION=ON                                                   ^
 -DMESH_MOVING_APPLICATION=ON                                                    ^
--DSOLID_MECHANICS_APPLICATION=ON                                                ^
--DCONSTITUTIVE_MODELS_APPLICATION=ON                                            ^
--DDELAUNAY_MESHING_APPLICATION=ON                                               ^
--DCONTACT_MECHANICS_APPLICATION=ON                                              ^
--DPFEM_APPLICATION=ON                                                           ^
--DPFEM_SOLID_MECHANICS_APPLICATION=ON                                           ^
--DPFEM_FLUID_DYNAMICS_APPLICATION=ON                                            ^
```

Comment:
Unintented change?

Reply:
less than you think. But now I guess I should revert it.